recipes/mysql-connector-cpp/all/conandata.yml

```diff
@@ -0,0 +1,13 @@
+sources:
+  "8.3.0":
```

Comment:
Do you really need all these versions? Otherwise, I would ask to keep ONLY 8.3.0. People still can ask for older versions by demand.

Reply:
Ping @samuaz

Reply:
for me just latest, so lets remove the oldest versions